src/d/actor/d_a_steam_tag.cpp

```diff
@@ -4,56 +4,209 @@
 //
 
 #include "d/actor/d_a_steam_tag.h"
+#include "JSystem/J3DGraphBase/J3DSys.h"
+#include "JSystem/JParticle/JPAEmitter.h"
+#include "d/d_cc_d.h"
+#include "d/d_com_inf_game.h"
+#include "d/d_item.h"
+#include "d/d_item_data.h"
 #include "d/d_procname.h"
+#include "dolphin/mtx/vec.h"
+
+static dCcD_SrcCps l_cps_src = {
+    // dCcD_SrcGObjInf
+    {
+        /* Flags             */ 0,
+        /* SrcObjAt  Type    */ AT_TYPE_WIND,
+        /* SrcObjAt  Atp     */ 0,
+        /* SrcObjAt  SPrm    */ cCcD_AtSPrm_Set_e | cCcD_AtSPrm_GrpAll_e,
+        /* SrcObjTg  Type    */ 0,
+        /* SrcObjTg  SPrm    */ 0,
+        /* SrcObjCo  SPrm    */ 0,
+        /* SrcGObjAt Se      */ 0,
+        /* SrcGObjAt HitMark */ 0,
+        /* SrcGObjAt Spl     */ 0,
+        /* SrcGObjAt Mtrl    */ 0,
+        /* SrcGObjAt SPrm    */ dCcG_AtSPrm_NoHitMark_e,
+        /* SrcGObjTg Se      */ 0,
+        /* SrcGObjTg HitMark */ 0,
+        /* SrcGObjTg Spl     */ 0,
+        /* SrcGObjTg Mtrl    */ 0,
+        /* SrcGObjTg SPrm    */ dCcG_TgSPrm_NoHitMark_e,
+        /* SrcGObjCo SPrm    */ 0,
+    },
+    // cM3dGCpsS
+    {
+        /* P0 */ 0.0f, 0.0f, 0.0f,
+        /* P1 */ 0.0f, 0.0f, 0.0f,
+        /* Height */ 200.0f,
+    },
+};
+
+daSteamTag_mData daSteamTag_c::mData = {15861, 49807, 300, 100, 50, 150, 0xB4, 0x00, 0x00};
```

Comment:
For struct initializers I usually like to put each field on a separate line and add a comment with the name of the field (like `l_cps_src` has above) because it makes it more clear to someone reading the code where certain values are defined (such as these timer ranges here)